draft_ui.py

```diff
@@ -1,28 +1,21 @@
 from typing import Optional
+from talon import Context
+from talon.experimental.textarea import TextArea, Span, DarkThemeLabels
```

Comment:
It looks like the entire patch for this file is reformatting, maybe just remove it from the PR?

Reply:
Can do